### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by BillyJW**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+        <th>
+        <time>14:11</time></th>
+        <th></th>
+      </tr>
+      <tr>
+        <td></td>
+        <td>
+          <p>
+            looking up complementry colors, discovered i would make for a poor
+             designer based on color opions alone, i am fine with this discovery
```
Comment (situlillian) [inline 02week/blog/index.html:235]: 💯 self awareness is the first step

File: 02week/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+
+    </table>
+
+    </h1>
+  </body>
+  <footer>
+    <ul class="footer">
+      <li>
+        <a href="../bio/index.html">Developed by Billy Watson</a>
+        <!-- dont ask why, it's just better off this way -->
```
Comment (situlillian) [inline 02week/blog/index.html:389]: good use of comments

File: 02week/blog/css/style.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+table {
+  margin-left: 30%;
+  margin-right: 30%;
+  table-layout: auto;
+  text-align: left;
+  /* color: #ffffff; */
+}
+
+table, th, td {
+    /* border: 1px solid black; */
```
Comment (situlillian) [inline 02week/blog/css/style.css:26]: when you're ready to finalize your project, I would suggest removing all the commented out code

**Review by situlillian — APPROVED** [review]
Like the use of tables; makes everything more formatted.

Looks good. Go ahead and merge.

**Comment by DinhH** [comment]
Also, can you make me a collaborator on this repository? Navigate to your web-workbook repository on github & at the top (not tippy top) of the page there is a settings tab. Under there you will see a button on the left called collaborators. Add all of us, dinhh, situlillian and shelbyinloes

**Pull request merged**